worker/src/RTC/NackGenerator.cpp

```diff
@@ -277,7 +297,10 @@ namespace RTC
 				continue;
 			}
 
-			if (filter == NackFilter::TIME && (nackInfo.sentAtMs == 0 || nowMs - nackInfo.sentAtMs >= this->rtt))
+			if (
+			  filter == NackFilter::TIME &&
+			  (nackInfo.sentAtMs == 0 ||
+			   nowMs - nackInfo.sentAtMs >= (this->rtt > 0u ? this->rtt : DefaultRtt)))
```

Comment:
```suggestion
			   nowMs - nackInfo.sentAtMs >= (this->rtt > 0.0f ? this->rtt : DefaultRtt)))
```

Reply:
Nope. `this->rtt` in `NackGenerator` is `uint32_t`.